### DISCUSSION TITLE
Policies are affecting generator wrongly

### DISCUSSION THREAD

**Discussion by lifeofguenter** [dani-garcia/vaultwarden#4413] [Q&A]
Only on Desktop + Browser extension I get:

<img width="367" alt="Screenshot 2024-03-08 at 07 33 36" src="https://github.com/dani-garcia/vaultwarden/assets/241073/b297c497-0be1-467a-8962-80bcd659436c">

But over the web-ui I get the correct policy:

<img width="753" alt="Screenshot 2024-03-08 at 07 34 14" src="https://github.com/dani-garcia/vaultwarden/assets/241073/865d5ba1-c85b-45bc-9726-e19633a0a185">

**Comment by stefan0xC** [comment]
Cf. https://github.com/dani-garcia/vaultwarden/discussions/4300#discussioncomment-8328176
